transport-native-epoll/src/main/java/io/netty/channel/epoll/AcceptedSocket.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.epoll;
+
+final class AcceptedSocket {
```

Comment:
Sure.  I don't think we will be able to stuff anything else in so that is fine.  I'm assuming we can't directly pass the byte[] array through JNI?